src/pages/tutorials/introduction-to-layer-types.md

```diff
@@ -8,4 +8,89 @@ layout: tutorials.hbs
+L.esri.featureLayer({
+	url: 'http://sampleserver3.arcgisonline.com/ArcGIS/rest/services/BloomfieldHillsMichigan/LandusePlanning/FeatureServer/0'
+})
+
+// note the MapServer part of this url - that means this layer is coming from a MapServer service 
+// To use a MapServer as a featureLayer you also have to specify a layer index at the end of the url, in this case '0' 
+L.esri.featureLayer({
+	url: 'http://sampleserver1.arcgisonline.com/ArcGIS/rest/services/Demographics/ESRI_Population_World/MapServer/0'
+})
+```
+
+If you want to edit data in the browser then you will need to be consuming a `FeatureServer` service.
+
+### What does the data look like?
+When you request a feature layer from an ArcGIS Server the response might look something like 
+```js
+{"type":"FeatureCollection","crs":{"type":"name","properties":{"name":"EPSG:4326"}},
+ "features":[
+     {"type":"Feature",
+     "id":2,
+     "geometry":{"type":"Point","coordinates":[-122.647466,45.543523]},
+     "properties":{"FID":2,"GEODB_OID":144,"OBJECTID":144,"TREEID":164,"STATUS":"Heritage","SCIENTIFIC":"Platanus x acerifolia","COMMON_NAM":"London planetree","STATEID":"1N1E26AC  12000","ADDRESS":"1728 NE STANTON ST","HEIGHT":82,"SPREAD":85,"CIRCUMFERE":15.44,"DIAMETER":59,"YEAR":1997,"OWNER":"Private, Front Yard","NOTES":"south side between NE 17th Ave and NE 18th Ave"}
+     }
+  ]
+}
+```
+You can see in the above sample that we have some location information in the `coordinates` array as well as the feature properties in the `properties` object.
+
+
+## Dynamic Map Layers
+A [Dynamic Map Layer](http://esri.github.io/esri-leaflet/api-reference/layers/dynamic-map-layer.html) involves pulling down a representation of your data from an ArcGIS for Server instance to your web app, this typically means an image such as a jpg or a png. Esri-Leaflet then takes care of placing the image on the correct part of the map for you.
+
+The benefit of a Dynamic Map Layer is that only the final representation of the data is sent to the client in the form of an image, this is hugely beneficial when dealing with large datasets with complex geometries. Rather than sending megabytes of data a simple image can be sent instead. Dynamic Map Layers are also handy because the images are rendered on the fly by the server which is beneficial if your dataset is constantly changing. 
+
+The downside of a Dynamic Map Layer is that if you require additional information about a feature (such as attribute information to generate a popup) then you'll need to make an additional request to the browser.
+
+### How to consume Dynamic Map Layers from ArcGIS for Server
+Dynamic Map Layers can be created in Esri-Leaflet from a `MapServer` service published by ArcGIS for Server. It utilises the Query Operation from the Dynamic Layer part of the [ArcGIS for Server Rest API](http://resources.arcgis.com/en/help/rest/apiref/).
```

Comment:
dynamicMapLayers actually utilize the [/exportMap](http://resources.arcgis.com/en/help/arcgis-rest-api/index.html#/Export_Map/02r3000000v7000000/) operation